src/client/telemetry/index.ts

```diff
@@ -1222,6 +1222,15 @@ export interface IEventNamePropertyMapping {
          */
         failed: boolean;
     };
+    /**
+     * Telemetry event sent when `python.terminal.activateEnvInCurrentTerminal` setting is set to `true` and an active terminal is present
+     */
+    [EventName.ACTIVATE_ENV_IN_CURRENT_TERMINAL]: {
+        /**
+         * The name of terminal being activated
```

Comment:
With the missing period and some clarification:

```suggestion
         * The name of terminal in which the Python environment was activated.
```